tests/integration/test_sms_endpoint_credendials_formatting.py

```diff
@@ -0,0 +1,23 @@
+from sinch.domains.sms.endpoints.sms_endpoint import SMSEndpoint
+from sinch.core.enums import HTTPAuthentication
+
+
+def test_sms_endpoint_service_plan_id_credentials_processing(sinch_client_sync_with_service_plan_id, service_plan_id):
+    sms_endpoint = SMSEndpoint(
+        sinch=sinch_client_sync_with_service_plan_id,
+        request_data={}
+    )
+    assert sms_endpoint.project_or_service_id == service_plan_id
+    assert sms_endpoint.HTTP_AUTHENTICATION == HTTPAuthentication.SMS_TOKEN.value
```

Comment:
Why do you need to set this values manually while the SMS endpoint is initialized with the right sinch client?
Same question for the test below.